apps/docs-snippets2/src/cookbook/resubmitting-failed-transactions/submitting.ts

```diff
@@ -0,0 +1,25 @@
+// #region resubmitting-failed-transactions-1
+import { Provider, Wallet } from 'fuels';
+import { LOCAL_NETWORK_URL, WALLET_PVT_KEY } from 'src/env';
+
+const provider = await Provider.create(LOCAL_NETWORK_URL);
+const wallet = Wallet.fromPrivateKey(WALLET_PVT_KEY, provider);
+
+const baseAssetId = provider.getBaseAssetId();
+
+const transferAmount = 1000;
+
+const transactionRequest = await wallet.createTransfer(
+  wallet.address,
+  transferAmount,
+  baseAssetId
+);
+
+const response = await wallet.sendTransaction(transactionRequest);
+// #endregion resubmitting-failed-transactions-1
+
+// #region resubmitting-failed-transactions-2
```

Comment:
This is relatively minor but just for consistency this `waitForResult` is isolated from the rest of the snippet and thus not a pastable snippet within itself. Perhaps we should move it to the region above?

Reply:
@maschad It is like that because of how the doc page instructions a laid out.
<img width="613" alt="Screenshot 2024-11-12 at 10 45 14 AM" src="https://github.com/user-attachments/assets/ed248e71-ba49-430e-ae5f-bc0e91b28b8c">

Reply:
Perhaps we can make an exception here, let's see how users feel.